service/internal/bootstrap/clusternamespace.go

```diff
@@ -28,7 +28,7 @@ func (b *Bootstrap) createReadClusterNamespaceAppsRole(ctx context.Context) erro
 			Name: key.ReadClusterNamespaceAppsRole,
 			Labels: map[string]string{
 				label.ManagedBy:              project.Name(),
-				label.DisplayInUserInterface: "true",
+				label.DisplayInUserInterface: "false",
```

Comment:
These roles are bound automatically so I don't think we need to show them. However, if there is a reason to do it I can also change it back.